File: src/main/scala/io/kamon/meta/reporter/KanelaInfoProvider.scala
package io.kamon.meta.reporter

trait KanelaInfoProvider
object KanelaInfoProvider {
  def getRecorded: java.util.Map[String, java.util.Map[String, java.util.List[String]]] = java.util.Collections.EMPTY_MAP.asInstanceOf[java.util.Map[String, java.util.Map[String, java.util.List[String]]]]

  def getErrors: java.util.Map[String, java.util.List[Throwable]] = java.util.Collections.EMPTY_MAP.asInstanceOf[java.util.Map[String, java.util.List[Throwable]]]
}

File: src/main/java/io/kamon/meta/reporter/KanelaInfoProvider.java
package io.kamon.meta.reporter;

import java.util.Collections;
import java.util.Map;

public class KanelaInfoProvider {
        public static final KanelaInfoProvider MODULE$ = null;

        public Boolean isActive() {
            return false;
        }

        public Map<String, String> modules() {
            return Collections.emptyMap();
        }

        public Map<String, java.util.List<Throwable>> errors() {
            return Collections.emptyMap();
        }


}